packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -340,14 +365,28 @@ describe('holocronModule', () => {
     const reducer = () => fromJS({ x: 3 });
     const mergeProps = (stateProps, dispatchProps, ownProps) => {
       const xy = stateProps.moduleState.x * ownProps.y;
-      return Object.assign({}, stateProps, dispatchProps, ownProps, { xy });
+      return {
+        ...stateProps, ...dispatchProps, ...ownProps, xy,
```

Comment:
trying something new, by suggestion of @JAdshead & @mtomcal so 🤞
```suggestion
        ...stateProps,
        ...dispatchProps,
        ...ownProps, 
        xy,
```

Reply:
I'm leaning towards the first one as you can see all the props in one line

Reply:
I find the separate lines easier to follow